### PR TITLE
fix(webapp): fix `viewport``width=device-width` by removing it

### LINKED ISSUE #5627: 🐛 [Bug] Responsive View on iPhone8
## 🐛 Bugreport

Responsive view of iphone8 is not properly working.

Ressources:
https://stackoverflow.com/questions/46313640/iphone-x-8-8-plus-css-media-queries

### PR DESCRIPTION
**Pull request by Tirokk**
## 🍰 Pullrequest

Fix `viewport``width=device-width` by removing it, because it does not work correctly for Safari on iOS.

### Issues

- fixes #5627

### Todo

- [ ] test it on Google Android

### CODE REVIEW

File: webapp/nuxt.config.js [hunk 1]
```diff
@@ -62,7 +62,8 @@ export default {
       },
       {
         name: 'viewport',
-        content: 'width=device-width, initial-scale=1',
+        // content: 'width=device-width, initial-scale=1', // "width=device-width" does not work correctly for Safari on iOS, see https://stackoverflow.com/a/37136777
```
Comment (Elweyn) [inline webapp/nuxt.config.js:65]: Git history has this stored, please remove comment.

File: webapp/maintenance/source/nuxt.config.maintenance.js [hunk 2]
```diff
@@ -15,7 +15,8 @@ export default {
       },
       {
         name: 'viewport',
-        content: 'width=device-width, initial-scale=1',
+        // content: 'width=device-width, initial-scale=1', // "width=device-width" does not work correctly for Safari on iOS, see https://stackoverflow.com/a/37136777
```
Comment (Elweyn) [inline webapp/maintenance/source/nuxt.config.maintenance.js:18]: Same

**Review by Elweyn — COMMENTED** [review]
You can set the stockoverflow link in the commit message.

File: webapp/maintenance/source/nuxt.config.maintenance.js (hunk 2 above)
Comment (Tirokk) [reply webapp/maintenance/source/nuxt.config.maintenance.js]: Done

File: webapp/nuxt.config.js (hunk 1 above)
Comment (Tirokk) [reply webapp/nuxt.config.js]: Done

**Review by Elweyn — APPROVED** [review]
Tested on Safari, Brave & mobile-devices of brave

**Pull request merged**